### PR TITLE
1 Solving issue #3533

### PR DESCRIPTION
**Pull request by Jenniline**
Adding a description on setting up a new twitter application for users.
The description informs them that the process is lengthy so they don't get disappointed.
I have made the edit to the en.json file as described in the problem description.

This pull request makes the following changes:
-Adding a description on setting up a new twitter application for users.
-The description informs them that the process is lengthy so they don't get disappointed.

Testing checklist:
- [ ]

- [ ] I certify that I ran my checklist

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

**Comment by Erioldoesdesign** [comment]
Hey @Jenniline thank you for taking on this issue! It's great to see you committing to the repo again ❤️ 

So you have:

"intro_step_1": "Step 1: Create a new Twitter application",
"intro_step_1": "Step 1: Create a new Twitter application: Twitter applications may take a few hours or a few days to be approved by Twitter. Please be aware of this if you need this data quickly.",

Can we change the 2nd "intro_step_1":  to "intro_step_2":  and the the visible text to Step 2: 

I also think we can shorten the copy to "Twitter applications may take a few hours or ~a few~ days to be approved by Twitter. Please be aware of this if you need this data quickly.

So just removing the second 'a few'

Great work!

**Comment by Jenniline** [comment]
Hi @Erioldoesdesign if you look keenly at my Pull request I have completely erased the line 

"intro_step_1": "Step 1: Create a new Twitter application", 
and I have replaced it with 
intro_step_1": "Step 1: Create a new Twitter application: Twitter applications may take a few hours or a few days to be approved by Twitter. Please be aware of this if you need this data quickly.",

If I do what you are suggesting there might not be chronology in the workflow of the twitter section because  intro_step_2 might appear twice since it is already written at the point  
"intro_step_2": "Step 2: Generate a consumer key and secret",
Except you want me to insert "intro_step_3" that I can do

But for now
 I have cancelled the words "a few" as you requested and I have and left the description for the lengthy process. 

I have made another pull request for the issue with the changes I have made. @Erioldoesdesign and @rowasc please review and let me know.

Thanks

**Comment by Erioldoesdesign** [comment]
Ah thanks Jenniline! I completely overlooked the red 'deleted' line 🤦‍♂ my bad! thanks for this/the new pull request~

**Comment by Angamanga** [comment]
@Jenniline It looks good but the "few days" is still there in the code, you need to make a commit into this branch ("patch-1") in order for it to show up.

**Comment by Jenniline** [comment]
Hi, Anna I have made a commit to "patch-1" with the changes you requested. here is the link 
https://github.com/Jenniline/platform-client/commit/97bb92b3052208ccde238c89619551a0aa84ac77

Please review.

Thanks 
Jenniline

File: app/common/locales/en.json [hunk 1]
```diff
@@ -1119,7 +1119,7 @@
                 }
             },
             "twitter": {
-                "intro_step_1": "Step 1: Create a new Twitter application",
+                "intro_step_1": "Step 1: Create a new Twitter application: <br />Twitter applications may take a few hours or days to be approved by Twitter.<br />Please be aware of this if you need this data quickly.",
```
Comment (Angamanga) [inline app/common/locales/en.json]: @Jenniline One more thing, could you remove the " in the end of the sentence?

Comment (Angamanga) [reply app/common/locales/en.json]: Nevermind, I missed there are a " in the beginning, forget my comment! Merging! Thank you for your contribution! 🎉

**Pull request merged**